src/components/Error/Error.test.tsx

```diff
@@ -1,26 +1,23 @@
-import React from 'react';
-import { render, screen } from '@testing-library/react';
-import { BrowserRouter } from 'react-router-dom';
-import Error from './Error';
-
-describe('Error', () => {
-  it('all element containers should render accordingly', () => {
-
+import React from "react";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { BrowserRouter } from "react-router-dom";
+import Error from "./Error";
+
+describe("Error", () => {
+  it("all element containers should render accordingly", () => {
     render(
       <BrowserRouter>
-        <Error 
-          errorMessage={'here is an error for you'}
-          errorNum={404}
-        />
+        <Error errorMessage={"here is an error for you"} errorNum={404} />
       </BrowserRouter>
-    )
+    );
```

Comment:
Was it your linter that changes all the single quotes to double quotes or did you do it manually?